test/automated.rb

```diff
@@ -0,0 +1,3 @@
+require_relative "test_init"
+
+TestBench::CLI.()
```

Comment:
this is stale, I'm not sure does it even work on new TestBench. I'm not able to test is since I can't install it locally.

Here you have a documentation:
https://github.com/test-bench/test-bench?tab=readme-ov-file#batch-runner

It should be something like:
```
require_relative './test_init'

TestBench::Run.(
  'test/automated',
  exclude: '{_*,*sketch*,*_init,*_tests}.rb'
) or exit(false)
```

Reply:
@arepnikov This is what we have in our other components for TestBench

Reply:
hm. I recall that I had some problems with TestBench::CLI.(), but since it works then leave it :)